docs/tutorial.rst

```diff
@@ -1577,6 +1589,12 @@ generations in the past, there were only two lineages left. The continuous
 time standard coalescent model was then used to simulate the ancient past of
 these two lineages.
 
+Note that migration rates between populations in the discrete-time
+Wright-Fisher model obey a sum constraint (see the :ref:`API
+<sec_api_simulation_model>`), whereas those in continuous-time models
+do not. Therefore migration rates may need to be modified simultaneously
+with transitions between simulation models.
+
```

Comment:
not sure if we need to say this caveat again, as usually people will not have migration rates that are so high?